crates/pg_configuration/src/database.rs

```diff
@@ -36,15 +40,7 @@ impl Default for DatabaseConfiguration {
             username: "postgres".to_string(),
             password: "postgres".to_string(),
             database: "postgres".to_string(),
+            conn_timeout_secs: Some(10),
```

Comment:
10 seconds is a long time :D usually the connection will happen either within <1s or never, maybe we should set this a bit lower to report issues earlier?

Reply:
I was thinking 2s maybe

Reply:
Hmm are you sure? The default SQLX timeout is 30s, and the default mongo driver timeout is 10s. I would expect that they choose sane defaults. Also, what if connection is not done locally, but e.g. from a GH Action Runner to an org's DB, or from a local VS Code to a remote DB?  

I'm happy to go with the 1-2s, just challenging. :)

Reply:
Mh yeah good point. I am just worried that users might wonder what's wrong when they won't see any info for 10s. But we can also show something to the user somehow.